tests/test_parser.py

```diff
@@ -691,13 +691,13 @@ def test_parse_boolean(text: str, expected_value: bool | None):
         ("-1", -1, False, True),
         ("-1", -1, True, True),
         ("true", None, False, False),
-        ("true", 1, True, False),
+        ("true", True, True, False),
```

Comment:
nit: I'm not sure how I feel to rely on that to be honest.
I would rather just explicitely say that `true` is `1` and `false` is `0`, rather than passing
by the boolean-int Python conversion?

Reply:
1 == True in Python, so this just felt prettier

Reply:
I'm happy to revert if you'd like